### DISCUSSION TITLE
Meet The Signal New API

### DISCUSSION THREAD

**Discussion by alimd** [AliMD/alwatr#90] [Announcements]
https://github.com/AliMD/alwatr/tree/next/package/signal#readme

**Comment by alimd** [comment]
Waiting for final review #88

**Comment by MM25Zamanian** [comment]
`interface VatrSignals` ???
Please Change

**Reply by alimd** [reply]
تو فایل اصلاح شده

**Comment by alimd** [comment]
Breaking Change: New Listener API Interface 🌹
